Makefile

```diff
@@ -12,13 +16,18 @@ test: check-gopath
 	go test -timeout 3m -tags "${BUILDTAGS}" ${TESTFLAGS} -v ./...
 	go test -timeout 3m ${TESTFLAGS} -v ./...
 
+${GOLANGCI_LINT}:
+	export \
+		VERSION=v1.23.6 \
```

Comment:
1. `1.23.7`
2. I think `golangci-lint` installation belongs to `.travis.yml` rather than Makefile. For example, I do have it installed on my system already, and won't need an extra copy for this package when running `make lint`.

Reply:
> 1. `1.23.7`

Updated it to the latest version.

> 2. I think `golangci-lint` installation belongs to `.travis.yml` rather than Makefile. For example, I do have it installed on my system already, and won't need an extra copy for this package when running `make lint`.

I disagree, because this would lead into inconsistent results depending on the machine we're running `make lint`. I would prefer in always running the same version of golangci-lint, independently if on CI or locally. WDYT?